### ISSUE TITLE
Horizontal-scroll flippers should only show on hover

### ISSUE THREAD

**Issue by robarbms** [microsoft/fluentui#18181]
#### Describe the feature that you would like added

Flippers on the horizontal-scroll component should be hidden by default and only show when hovered.

#### What component or utility would this be added to

Horizontal-scroll component.

#### Have you discussed this feature with our team, and if so, who

@chrisdholt and Karim Raphael

**Comment by chrisdholt** [comment]
@robarbms do you want to take this as a CSS change here? I think it would be great if FAST could provide an attribute to support instances where you can toggle the visibility AND bring the flippers into the tab order. This would only be necessary in instances where content within the horizontal scroll is *not* actionable, for example a bunch of photos, etc.

**Comment by robarbms** [comment]
> @robarbms do you want to take this as a CSS change here? I think it would be great if FAST could provide an attribute to support instances where you can toggle the visibility AND bring the flippers into the tab order. This would only be necessary in instances where content within the horizontal scroll is _not_ actionable, for example a bunch of photos, etc.

I can create a separate issue in the FAST repo to add an attribute for toggling visibility.

**Comment by chrisdholt** [comment]
@robarbms that would be great - I see the PR that closes this already, awesome!
